runelite-client/src/main/java/net/runelite/client/plugins/gpu/GpuPlugin.java

```diff
@@ -1195,26 +1241,17 @@ private void uploadScene()
 		IntBuffer vertexBuffer = this.vertexBuffer.getBuffer();
 		FloatBuffer uvBuffer = this.uvBuffer.getBuffer();
 
-		if (bufferId != -1)
-		{
-			GLUtil.glDeleteBuffer(gl, bufferId);
-			bufferId = -1;
-		}
-
-		if (uvBufferId != -1)
-		{
-			GLUtil.glDeleteBuffer(gl, uvBufferId);
-			uvBufferId = -1;
-		}
+		if (bufferId == -1)
+			bufferId = glGenBuffers(gl);
 
-		bufferId = glGenBuffers(gl);
-		uvBufferId = glGenBuffers(gl);
+		if (uvBufferId == -1)
+			uvBufferId = glGenBuffers(gl);
 
 		gl.glBindBuffer(gl.GL_ARRAY_BUFFER, bufferId);
-		gl.glBufferData(gl.GL_ARRAY_BUFFER, vertexBuffer.limit() * Integer.BYTES, vertexBuffer, gl.GL_STATIC_COPY);
+		gl.glBufferData(gl.GL_ARRAY_BUFFER, vertexBuffer.limit() * Integer.BYTES, vertexBuffer, gl.GL_DYNAMIC_DRAW);
```

Comment:
The scene buffers are not updated per frame nor even drawn from directly, instead they are (readonly) accessed via ssbo in the compute shaders.